citesphere/src/main/java/edu/asu/diging/citesphere/core/authority/impl/ViafAuthorityImporter.java

```diff
@@ -94,12 +123,65 @@ public String getId() {
     }
 
     @Override
-    public AuthoritySearchResult searchAuthorities(String firstName, String lastName, int page, int pageSize) {
-        throw new UnsupportedOperationException();
+    public AuthoritySearchResult searchAuthorities(String firstName, String lastName, int page, int pageSize)
+            throws AuthorityServiceConnectionException {
+
+        pageSize = Math.min(10, pageSize);
```

Comment:
why doesn't this take page sizes bigger than 10? if there is a reason, there should be a comment.